sources/packages/backend/package.json

```diff
@@ -25,13 +25,16 @@
     "test:e2e:api": "npm run db:seed:test filter=CreateInstitutionsAndAuthenticationUsers && cross-env ENVIRONMENT=test jest --collect-coverage --verbose --config ./apps/api/test/jest-e2e.json --forceExit",
     "test:e2e:api:local": "cross-env ENVIRONMENT=test jest --config ./apps/api/test/jest-e2e.json --forceExit",
     "test:e2e:workers": "npm run migration:run && cross-env ENVIRONMENT=test jest --collect-coverage --verbose --config ./apps/workers/test/jest-e2e.json --forceExit",
+    "test:e2e:workflow:local": "cross-env ENVIRONMENT=test jest --verbose --config ./workflow/test/jest-e2e.json --forceExit",
+    "test:e2e:workflow": "npm run deploy:camunda:definitions && cross-env ENVIRONMENT=test jest --verbose --config ./workflow/test/jest-e2e.json --forceExit",
     "test:e2e:queue-consumers": "npm run migration:run && cross-env ENVIRONMENT=test jest --collect-coverage --verbose --config ./apps/queue-consumers/test/jest-e2e.json --forceExit",
     "migration:run": "node --abort-on-uncaught-exception --require ts-node/register apps/db-migrations/src/main.ts",
     "migration:create": "cd apps/db-migrations/src/migrations && npx typeorm-ts-node-commonjs migration:create",
     "migration:revert": "npx typeorm-ts-node-commonjs migration:revert -d apps/db-migrations/src/data-source.ts",
     "db:seed:test": "npm run build test-db-seeding && npm run migration:run && node dist/apps/test-db-seeding/main.js",
     "//db:seed:test:clean": "Drops the database and all its data. Be careful with this command on production since this command will erase all your database tables and their data.",
-    "db:seed:test:clean": "npm run build test-db-seeding && node dist/apps/test-db-seeding/main.js task=clean-db"
+    "db:seed:test:clean": "npm run build test-db-seeding && node dist/apps/test-db-seeding/main.js task=clean-db",
+    "deploy:camunda:definitions": "ts-node ./workflow/src/deploy.ts"
```

Comment:
Not PR related but I have the feeling that we should bring the forms also in here. Even with the forms being UI/backend, the deploying itself is a backend concern, and also the form definitions are used for backend validations.
I believe that makes more sense to have here also one `deploy:forms:definitions`.
We we have an agreement we can create a ticket for it.